app/models/issue.rb

```diff
@@ -1,8 +1,10 @@
 class Issue < ApplicationRecord
+  belongs_to :user
+  attr_accessor :attachment
```

Comment:
why removed? you need to keep this file. How later you will generate link to it?
you need to add that field again